docs/index.md

```diff
@@ -208,7 +214,39 @@ For example, you can start the latest version of `relay` as follows:
 ❯ docker run -v $(pwd)/configs/:/work/.relay/ getsentry/relay run
 ```
 
-This example command assumes that Relay's configuration (`config.yml` and `credentials.json`) are stored in `./configs/` directory on the host machine.
+This example command assumes that Relay's configuration (`config.yml` and `credentials.json`) are stored in `./configs/` 
+directory on the host machine.
+
+
+## Logging and healthcheck
+
+Now you have a running relay, you might have noticed that relay displays some `INFO` messages,
+including:
+```sh
+INFO  relay::setup >   log level: INFO
+```
+
+This is the default logging level you can change this to show more or less info. 
+For details about configuring logging please see _[logging]_ on the options page.
+
+Relay provides two urls for health check and monitoring the live status:
+
+In our example relay is running at "http://localhost:3000" so the following urls are
+going to return a status check JSON file.
+
+The `GET http://localhost:3000/api/relay/healthcheck/live/` is used to test if Relay
+is running and able to respond to http requests.
+
+The `GET http://localhost:3000/api/relay/healthcheck/ready/` can be used to check if 
+Relay is running and has been able to authenticate with the upstream.
+
+In case of success both endpoints return:
```

Comment:
```suggestion
Relay provides two URLs for checking liveness and readyness of the service:

- `GET /api/relay/healthcheck/live/`: Tests if Relay is running and listening to HTTP requests.
- `GET /api/relay/healthcheck/ready/`: Tests if Relay is authenticated with the upstream and
  operating normally.

In case of success, both endpoints return a _200 OK_ response:
```